### PR TITLE
Fix links and list both API specs in README

### PR DESCRIPTION
**Pull request by mkungla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,32 @@
...
-* [WebVR Specification](https://w3c.github.io/webvr/)
-* [WebVR Charter](https://w3c.github.io/webvr/charter/)
+* [WebXR Device API Specification](https://immersive-web.github.io/webxr/)
+* [Legacy WebVR API Specification](https://immersive-web.github.io/webvr/)
+* [WebVR Charter](https://immersive-web.github.io/webxr/charter/)
 
 
 ## Communication
 
 * [W3C Community Group: `webvr`](http://www.w3.org/community/webvr/)
```
Comment (cvan) [inline README.md]: we ought to update this (cc @toji @NellWaliczek)

Comment (cvan) [reply README.md]: can remove the ": `webvr`" part to be consistent with the [usage above](https://github.com/immersive-web/webxr/pull/309/files#diff-04c6e90faac2675aa89e2176d2eec7d8R20)

File: README.md [hunk 2]
```diff
@@ -2,30 +2,32 @@
...
+This is currently the repository for the [WebVR Community Group](https://www.w3.org/community/webvr/) and the [WebXR Specification](https://immersive-web.github.io/webxr/). We have a [charter](https://immersive-web.github.io/webxr/charter/) (in progress).
 
 The specification has recently undergone a name change, so expect to see multiple references to "WebVR" sprinkled throughout.
 
 
 ## Specifications
 
-* [WebVR API Specification](https://w3c.github.io/webvr/): main specification for JavaScript API for accessing VR displays.
+* [WebXR Device API Specification](https://immersive-web.github.io/webxr/): main specification for JavaScript API for accessing VR and AR devices, including sensors and head-mounted displays.
+* [Legacy WebVR API Specification](https://immersive-web.github.io/webvr/) Legacy WebVR API 1.1 specification for JavaScript API for accessing VR displays. Development of the WebVR API has halted in favor of being replaced the WebXR Device API. Several browsers will continue to support this version of the API in the meantime.
```
Comment (cvan) [inline README.md]: `Legacy` -> `: Legacy`

File: README.md [hunk 3]
```diff
@@ -2,30 +2,32 @@
...
-This is currently the repository for the [WebVR Community Group](https://www.w3.org/community/webvr/) and the [WebXR Specification](https://immersive-web.github.io/webxr/). We have a [charter](https://w3c.github.io/webvr/charter/) (in progress).
+This is currently the repository for the [WebVR Community Group](https://www.w3.org/community/webvr/) and the [WebXR Specification](https://immersive-web.github.io/webxr/). We have a [charter](https://immersive-web.github.io/webxr/charter/) (in progress).
 
 The specification has recently undergone a name change, so expect to see multiple references to "WebVR" sprinkled throughout.
 
 
 ## Specifications
 
-* [WebVR API Specification](https://w3c.github.io/webvr/): main specification for JavaScript API for accessing VR displays.
+* [WebXR Device API Specification](https://immersive-web.github.io/webxr/): main specification for JavaScript API for accessing VR and AR devices, including sensors and head-mounted displays.
```
Comment (cvan) [inline README.md]: `:  main` -> `: Main`

File: README.md [hunk 4]
```diff
@@ -2,30 +2,32 @@
...
+* [Legacy WebVR API Specification](https://immersive-web.github.io/webvr/) Legacy WebVR API 1.1 specification for JavaScript API for accessing VR displays. Development of the WebVR API has halted in favor of being replaced the WebXR Device API. Several browsers will continue to support this version of the API in the meantime.
 * [Gamepad API Specification](https://w3c.github.io/gamepad/), introduces a low-level JS API interface for accessing gamepad devices.
```
Comment (cvan) [inline README.md]: `, introduces` -> `: Introduces`

File: README.md [hunk 5]
```diff
@@ -2,30 +2,32 @@
...
 * [Gamepad Extensions API Specification](https://w3c.github.io/gamepad/extensions.html): extends the Gamepad API to enable access to more advanced device capabilities.
```
Comment (cvan) [inline README.md]: `: extends` -> `: Extends`

**Comment by mkungla** [comment]
Updated first commit as requested, Having second look on it I found some inconsistency with links so I added another commit using reference-style link definitions.

**Comment by toji** [comment]
Thank you! This has been a messy move, so the help (and simply the extra pair of eyes) is appreciated!

**Pull request merged**